cranelift/module/src/data_context.rs

```diff
@@ -50,6 +50,8 @@ pub struct DataDescription {
     pub data_relocs: Vec<(CodeOffset, ir::GlobalValue, Addend)>,
     /// Object file section
     pub custom_segment_section: Option<(String, String)>,
+    /// Alignment
```

Comment:
Please document whether this is in bytes or in log2(bytes).

Reply:
Done